WordPress/WordPressStatsWidgets/LockScreenWidgets/LockScreenStatsWidget.swift

```diff
@@ -0,0 +1,36 @@
+import WidgetKit
+import SwiftUI
+
+@available(iOS 16.0, *)
+struct LockScreenStatsWidget<T: LockScreenStatsWidgetConfig>: Widget {
+    private var config: T!
```

Comment:
It looks like since `init()` implements `fatalError` then it can be a constant

```suggestion
    private let config: T
```

Reply:
Thanks, I don't know this!! 
Now the syntax looks makes sense 🙌